cot-cli/tests/migration_generator.rs

```diff
@@ -244,6 +244,48 @@ fn find_source_files() {
         .any(|f| f.file_name().unwrap() == nested_file_name));
 }
 
+#[test]
+fn list_migrations() {
+    let cot_cli_dir = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+    let example_dir = cot_cli_dir
+        .parent()
+        .unwrap()
+        .join("examples")
+        .join("todo-list");
```

Comment:
I don't think relying on examples in tests is a good idea. Tests should be as self-contained as possible, and it will be very weird if someone will modify or remove a random example and suddenly the tests will fail. Can we modify this so that the tests will create a temporary directory, put some migration files inside, and test `list_migrations` in such environment?